packages/electron/lib/electron.js

```diff
@@ -10,8 +10,12 @@ const paths = require('./paths')
 const install = require('./install')
 let fs = require('fs-extra')
 
+const debugStderr = require('debug')('cypress:internal-stderr')
```

Comment:
more of a nit but something like this?
```suggestion
const debugStderr = require('debug')('cypress:electron:internal:stderr')
```

Reply:
It's not limited to electron - if other third party libs write to stderr, they'll show up here too

Reply:
ah good point